app/controllers/api/v1/skills_controller.rb

```diff
@@ -0,0 +1,8 @@
+class Api::V1::SkillsController < Api::V1::BaseController
+  respond_to :json
+
+  def index
+    featured = params[:featured]
+    respond_with featured ? Skill.where(featured: featured) : Skill.all
```

Comment:
This should be extracted out. We have a `respond_with` that then is evaluating a ternary conditional. A good first start would be pulling the conditional out into a private method.